foo.rb

```diff
@@ -3,3 +3,5 @@ def bar
     "A very long string that is going to be flagged by Rubocop for being too long for its standards. But oh well :)"
   end
 end
+
+"Another bad piece of code"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.